test/linodes/linodes.bats

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bats
+
+load '../test_helper/bats-support/load'
+load '../test_helper/bats-assert/load'
+load '../common'
+
+@test "it should create linodes with a label" {
+    run linode-cli linodes create --type g6-standard-2 --region us-east --image $test_image --label cli-1 --root_pass $random_pass --text --delimiter "," --no-headers
+    assert_output --regexp ".*,cli-1,us-east,g6-standard-2,$test_image,.*"
+}
+
+@test "it should view the linode configuration" {
+    linode_id="$(linode-cli --text --no-headers linodes list | awk '{ print $1 }' | xargs)"
+    run linode-cli linodes view "$linode_id" --text --delimiter "," --no-headers
+    assert_output --regexp "$linode_id,cli-1,us-east,g6-standard-2,$test_image,.*"
+}
+
+@test "it should create a linode with the minimum required props" {
+    run linode-cli linodes create --type g6-standard-2 --region us-east --root_pass $random_pass
+    [ "$status" -eq 0 ]
+}
+
+# ONLY WORKS BECAUSE I HAVE A .linode-cli
+@test "it should fail to create a linode without a root_pass" {
+    run linode-cli linodes create --type g6-standard-2 --region us-east
+    [ "$status" -eq 1 ]
```

Comment:
if you included a `--text --no-headers` you could test for output like this (in addition to testing for the status):

```
Request failed: 400
root_pass	root_pass is required
```

Reply:
so, i tested this on prod last night, and realized that this is permissible on prod, but not staging? slightly confused, do i need to modify this in order for it to work across all envs?